interface/modules/custom_modules/oe-module-claimrev-connect/src/ClaimRevModuleSetup.php

```diff
@@ -0,0 +1,96 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    require_once "../../../../globals.php";
+
+    use OpenEMR\Modules\ClaimRevConnector\Bootstrap;
+
+    class ClaimRevModuleSetup
+    {
+        public function __construct() 
+        { } 
+        public static function DoesPartnerExists()
+        {
+            $x12Name = $GLOBALS['oe_claimrev_x12_partner_name'];
+            $sql = "SELECT * FROM x12_partners WHERE name = ?";
+            $sqlarr = array($x12Name);
+            $result = sqlStatementNoLog($sql,$sqlarr);  
```

Comment:
better to use sqlStatementNoLog unless grabbing sensitive data like password hashes

Reply:
I might be missing something obvious, I'm using sqlStatementNoLog on line 16. I'm confused with this one.

Reply:
I did remove an error_log I was doing that wasn't needed.

Reply:
We generally always use sqlStatement() since this then goes through the audit log in OpenEMR and will then be dependent on the audit log settings in globals.
We generally only use sqlStatementLog() when we don't want sensitive data like password hashes or encryption keys (if the sql query contains this sensitive information) to be stored in OpenEMR's audit log.